src/bindings/python/flux/wrapper.py

```diff
@@ -162,6 +162,12 @@ def build_argument_translation_list(self, fun_type):
     def __call__(self, calling_object, *args_in):
         calling_object.ffi.errno = 0
         caller = calling_object.handle
+        if self.add_handle and caller is None:
```

Comment:
This change seems to be somewhat independent of this commit?  I assume some bug was hit and you added it?  Perhaps needs a comment in the commit or should be in a separate commit?

Reply:
Oh. Good call.  That was added when I had a `destroy` method for `Future`.  But that method proved too tricky to handle consistency (too many corner cases), so I removed the method and then squashed the commit.  This stuck around.  I'll pull it out into its own commit (along with the relevant unit test, if there is one, if not, I'll make one)

Reply:
@chu11: I pulled this out into it's own commit and added a unit test in `t/python/wrapper.py` in the same commit.